scripts/travis-build-dependencies.sh

```diff
@@ -105,7 +110,7 @@ rm packages-required-master.tar.gz
 ## Install NautyTracesInterface in Travis
 if [ "$SETUP" == "travis" ] && [ "$NAUTY" != "no" ]; then
   echo -e "\nGetting master version of NautyTracesInterface"
-  git clone -b master --depth=1 https://github.com/gap-packages/NautyTracesInterface.git $GAPROOT/pkg/nautytraces
+  git clone -b master --depth=1 https://github.com/sebasguts/NautyTracesInterface.git $GAPROOT/pkg/nautytraces
```

Comment:
Not a huge problem, but could you change this back to `gap-packages` please?

To answer your question: I hadn't enabled Appveyor on this new repository yet, but I have done so now, so it should run in future (as is indeed happening over here https://github.com/digraphs/Digraphs/pull/347).

If you're ready for this to be merged (I'm happy to approve), then would you mind squashing the commits? I could do it with a squash and merge, but I'd feel happier to have all the CI (including Appveyor) run through fully once in the squashed form.

(Since approved reviews get dismissed after each push, I'll hold off approving till I actually merge).

Reply:
And can you change the occurrence of `gap-packages` in the file `scripts/travis-build-digraphs.sh` to be `digraphs`? I'm not going to insist that Appveyor works before merging this PR, but I'm pretty sure that if you make that change, then Appveyor test will pass (see https://ci.appveyor.com/project/wilfwilson/digraphs/builds/36359419 for an example)

Reply:
Will do! Was teaching all day, will fix first thing tomorrow or later tonight

Reply:
I think I made the requested changes. Hopefully everything will pass !